client/public/static/js/account/my_recipes.js

```diff
@@ -339,3 +359,167 @@ $(document).ready(function () {
   const myRecipesView = new MyRecipesView();
   myRecipesView.load();
 });
+
+async function updateRecipe(recipeName) {
+  const username = utils.getUserNameFromCookie();
+  if (!username) {
+    console.error(UNABLE_TO_UPDATE_USER_NOT_LOGGED_IN);
+    utils.showAjaxAlert("Error", UNABLE_TO_UPDATE_USER_NOT_LOGGED_IN);
+    return;
+  }
+
+  const userId = await utils.getUserIdFromUsername(username);
+  if (!userId) {
+    console.error(UNABLE_TO_UPDATE_USER_NOT_LOGGED_IN);
+    utils.showAjaxAlert("Error", UNABLE_TO_UPDATE_USER_NOT_LOGGED_IN);
+    return;
+  }
+
+  window.location = "/account/update_recipe?userRecipeName=" + recipeName;
```

Comment:
Change to be: 

`/account/update_recipe?userRecipeName=${recipeName}`

Reply:
I couldn't get this to work. RecipeName is not being seen as a variable in the string when done like this for me